suite-native/notifications/src/components/TransactionNotification.tsx

```diff
@@ -125,7 +125,6 @@ export const TransactionNotification = ({
                     transactionType={transactionType}
                     symbol={notification.symbol}
                     isAnimated={isIconAnimated}
-                    iconColor={isIconAnimated ? 'iconAlertYellow' : 'iconSubdued'}
```

Comment:
@shenkys explicitly asked for the icon to use the default colour no matter what. I missed this in the initial commit.